cel/library.go

```diff
@@ -281,6 +284,16 @@ func (stdLibrary) ProgramOptions() []ProgramOption {
 //
 // This is syntactic sugar for msg.elements[msg.elements.size()-1].
 
+// # Unwrap / UnwrapOpt
+//
+// Introduced in version: 2
+//
+// Returns a list of all the values that are not none in the input list of optional values.
+// Can be used as optional.unwrap(List[T]) or with postfix notation: List[T].unwrapOpt()
```

Comment:
super-nit: prefer lower case `list(T)` to `List[T]`

Could you also update the ext/README.md with the same content?

Reply:
Hm, but this is not an extension, just part of the standard library.

Reply:
So the other functions in this file like `hasValue()` `value()` etc are not documented in ext/README.md.
Should I create a category for them?

Reply:
Ah, my mistake. Eventually we should consolidate docs, but this can happen another time. Thanks!